src/Components/Artifacts/Hero/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from './Artifacts.module.css'
+
+
```

Comment:
ta bort en blankrad här med,  efter import:) (nu är jag superpetig, det är verkligen små detaljer)

Reply:
Thank you! Det är bra att vara petig, då blir koden läsbar :D 
Har tagit bort blankraderna samt lagt i bokstavordning :)